### ISSUE TITLE
No Data Showing

### ISSUE THREAD

**Issue by derekengel** [teelolws/Altoholic-Retail#95]
I installed the addon, and I have no data showing.  I have made sure to enable the datastore components.  I have no errors, but when I open the addon, it's just blank inside.  The datastore memory usage is all green but at 0kb.

**Comment by teelolws** [comment]
Show me some screenshots: your altoholic window, and your addons window

**Comment by derekengel** [comment]
I signed back into my account to get you the screenshots, and now I show data.  Very strange.  Originally I logged in with the addon enabled.  I opened my bank, guild bank, and professions.  I know it might not have been necessary, but I wanted to be thorough.  I even did a reload as well as logging off/on.  I did this on two characters.  I didn't show any data, so I opened this issue.  Now that I completely exited the game, is the only difference I could think of.
